hnn_core/gui/gui.py

```diff
@@ -911,12 +894,12 @@ def _get_evoked_widget(name, layout, style, location, data=None,
     if isinstance(data, dict):
         default_data.update(data)
     kwargs = dict(layout=layout, style=style)
-    mu = FloatText(value=default_data['mu'],
-                   description='Mean time:',
-                   **kwargs)
-    sigma = FloatText(value=default_data['sigma'],
-                      description='Std dev time:',
-                      **kwargs)
+    mu = BoundedFloatText(
+        value=default_data['mu'], description='Mean time:', min=0, max=1e6,
+        **kwargs)
+    sigma = BoundedFloatText(
+        value=default_data['sigma'], description='Std dev time:', min=0,
+        max=1e6, **kwargs)
```

Comment:
What's the step size here?

Reply:
the default value is 1 I think

Reply:
Let's set them to 0.01 for now as they're time related. In the future we can adjust further if needed.